### PR TITLE
Translation Follow Ups

### PR DESCRIPTION
**Pull request by mmcfarland**
* Fixes Save & Share title translation
* Fixes Measure tooltip and results bubble
* Fixes checkbox and continue button on InfoGraphic
* Sample spanish translation for region and measure plugin.

Connects #606 

Test:
* Change your region language to "es"
* See the following are translated:
    * See Save & Share title, 
    * Measure tooltip, in-progress template and final result
    * Identify Point infographic built-in text

screenshot from 2016-03-28 16 15 55
screenshot from 2016-03-28 15 17 33

### CODE REVIEW

File: src/GeositeFramework/locales/en.json [hunk 1]
```diff
@@ -53,5 +52,6 @@
     "Failed to return results within %f seconds": "Failed to return results within %f seconds",
     "No information is available for this location": "No information is available for this location",
     "Permalink:": "Permalink:",
-    "Don't Show This on Start": "Don't Show This on Start"
+    "Don't show this on start": "Don't show this on start",
+	"Continue": "Continue"
```
Comment (rajadain) [inline src/GeositeFramework/locales/en.json]: Spacing issue, prefer four spaces to a tab.

File: src/GeositeFramework/plugins/measure/locales/en.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
     "Measure": "Measure",
     "Measure distances and area on the map": "Measure distances and area on the map",
-    "Segment Length:": "Segment Length:",
-    "Total Length:": "Total Length:",
+    "Segment Length": "Segment Length",
+    "Total Length": "Total Length",
     "Area:": "Area:",
-    "Length:": "Length:"
+    "Length:": "Length:",
+	"Double-click to end a line segment. Single-click the first node to end a polygon.": "Double-click to end a line segment. Single-click the first node to end a polygon."
```
Comment (rajadain) [inline src/GeositeFramework/plugins/measure/locales/en.json]: Spacing issue, prefer four spaces to a tab.

File: src/GeositeFramework/plugins/measure/locales/es.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+     "Measure": "Medida",
+     "Measure distances and area on the map": "Medir distancias y el área en el mapa",
+     "Segment Length": "Duración del segmento",
+     "Total Length": "Longitud total",
+     "Area:": "Área de:",
+     "Length:": "Longitud:",
+	 "Double-click to end a line segment. Single-click the first node to end a polygon.": "Haga doble clic para terminar con un segmento de línea simple clic en el primer nodo de poner fin a un polígono."
```
Comment (rajadain) [inline src/GeositeFramework/plugins/measure/locales/es.json]: Spacing issue, prefer four spaces to a tab.

**Comment by rajadain** [comment]
+1 tested, the Save & Share was title was translated correctly, and the "Continue" button and checkboxes were as well. The measure plugin also worked correctly.

**Comment by rajadain** [comment]
Great job figuring out the root cause (ordering of the library load affecting translation)!

**Comment by mmcfarland** [comment]
Thanks.  I copied in the json from an online beautifier, I'll clean up the whitespace.

**Pull request merged**